opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/AnalyzerGuru.java

```diff
@@ -795,50 +326,55 @@ public static AnalyzerFactory findFactory(String factoryClassName)
                 // Ignore
             }
         }
-        
+
+        for (AnalyzerFactory factory : this.pluginFramework.getAnalyzersInfo().factories) {
+            if (factory.getClass().getSimpleName().equals(simpleName)) {
+                factoryClass = factory.getClass();
+                break;
+            }
+        }
+
         return factoryClass;
     }
-    
+
     /**
      * Find a {@code FileAnalyzerFactory} which is an instance of the specified
      * class. If one doesn't exist, create one and register it.
      *
      * @param factoryClass the factory class
      * @return a file analyzer factory
-     *
-     * @throws ClassCastException if the class is not a subclass of {@code
-     * FileAnalyzerFactory}
-     * @throws IllegalAccessException if the constructor cannot be accessed
-     * @throws InstantiationException if the class cannot be instantiated
-     * @throws NoSuchMethodException if no-argument constructor could not be found
+     * @throws ClassCastException        if the class is not a subclass of {@code
```

Comment:
Why are you aligning the description? I personally don't like it because if I were to add  `VeryLongLongLongLongException` then I would have to modify other places that have nothing to do with my changes which just makes the diff hard to read. That's the reason we don't do following:
```
int abc = 1;
int a   = 2;
int ab  = 3;
```

Reply:
Idea does that for all of javadoc stuff.

Descriptions and the actual code are two different things.